getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+    "\n",
+    "Otherwise, to ensure the user is not overwhelming the Azure services, make sure to wait between API calls when possible - this can be enforced by using the `time` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "99e45cb7-6e35-4420-bd41-651157e5596f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "time.sleep(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a85fe6-ee18-4fc8-88c2-b67297ac8ef8",
+   "metadata": {},
+   "source": [
+    "### Check the costs and token usage of a given model API call"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "561dfbfb-0933-4650-8230-23f1bc2b1f46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_community.callbacks.manager import get_openai_callback"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "id": "c9081591-0b58-44b3-89c7-c3a512f84f4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "messages = [\n",
+    "    SystemMessage(content=\"Translate the following from German into English\"),\n",
+    "    HumanMessage(content=\"What's the first planet in the Solar System?\"),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61c9a128-f702-4f48-b695-436629e5ac1a",
+   "metadata": {},
+   "source": [
+    "`get_openai_callback()` is a context manager of the [**OpenAICallbackHandler**](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/callbacks/openai_info.py) class, meaning it calls this class and creates an instance when used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "id": "178153e8-c37c-4ce2-bbb4-15ec6feb58f6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total tokens used: 37\n",
+      "Total prompt tokens: 27\n",
+      "Total prompt tokens: 10\n",
+      "Total cost (in dollars): $0.000285\n",
+      "Total successful requests): 1\n"
+     ]
+    }
+   ],
+   "source": [
+    "with get_openai_callback() as cb:\n",
+    "    model.invoke(messages)\n",
+    "    print(f\"Total tokens used: {cb.total_tokens}\")\n",
+    "    print(f\"Total prompt tokens: {cb.prompt_tokens}\")\n",
+    "    print(f\"Total prompt tokens: {cb.completion_tokens}\")\n",
+    "    print(f\"Total cost (in dollars): ${cb.total_cost}\")\n",
+    "    print(f\"Total successful requests): {cb.successful_requests}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab9705c6-0512-48b3-b117-8e1bc385cf86",
+   "metadata": {},
+   "source": [
+    "## Tools and Structured output"
```

Comment:
Do you think we should maybe narrow the focus of this section to only "how to use structured outputs" since that's mostly what's shown in our example? We could mention that this feature is useful with tools/function calling.

Reply:
Have updated the section to "how to use structured outputs" and added 'Tools' as a subheader below this.